dom-random-num/script.js

```diff
@@ -6,23 +6,53 @@ window.addEventListener("load", () => {
     // There are exceptions - like if the element isn't on the page yet because it gets added later
     
     // Instantiate the object needed for TODO #1 
```

Comment:
Hi there! I'm guessing you meant to open this PR on your own forked copy of this repo?